index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Hello World, we are software developers!</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="index.js"></script>
+    <nav id="navbar">
+      <a class="navlink" href="#welcome-section">About</a>
+      <a class="navlink" href="#projects">Work</a>
+      <a class="navlink" href="#profile">Profile</a>
+    </nav>
+    <div id="welcome-section">
+      <h1>Hey, i am Favour</h1>
+      <p class="wsp"><em>a software developer</em></p>
+    </div>
+
+    <div id="projects">
+      <h2 class="projects-header">My Projects</h2>
+
+      <div class="project-grid">
+        <a
+          class="projects project-title"
+          href="https://codepen.io/freeCodeCamp/full/zNqgVx"
+          target="_blank"
+        >
+          <img
+            class="images"
+            src="https://image.pbs.org/video-assets/eppEsWb-asset-mezzanine-16x9-LQ8tc0K.jpg.crop.1200x675.jpg"
+          />
+          <p class="project-ttle">Tribute Page</p>
+        </a>
+        <a
+          class="projects"
+          href="https://codepen.io/ritobrotomukherjee/details/xxpOQrq"
+          target="_blank"
+        >
+          <img
+            class="images"
+            src="https://image.pbs.org/video-assets/eppEsWb-asset-mezzanine-16x9-LQ8tc0K.jpg.crop.1200x675.jpg"
+          />
+          <p class="project-ttle">Tribute Page</p>
+        </a>
+        <a
+          class="projects"
+          href="https://codepen.io/ritobrotomukherjee/details/xxpOQrq"
+          target="_blank"
+        >
+          <img
+            class="images"
+            src="https://image.pbs.org/video-assets/eppEsWb-asset-mezzanine-16x9-LQ8tc0K.jpg.crop.1200x675.jpg"
+          />
+          <p class="project-ttle">Tribute Page</p>
+        </a>
+      </div>
+    </div>
```

Comment:
I think it will be better to wrap your `project-container` in a `<section>`  tag than in a `<div>` wrapper as it defines the section of websites. 

Also, `<img>` should always have an alt attribute (usually specifies the alternate text for an image) as  this is useful when the image is not displayed. [more info](https://www.w3schools.com/tags/att_img_alt.aspl)

This is an example of what a proper `<img>` tag should look like.
```
<img src="images/dinosaur.jpg"
     alt="The head and torso of a dinosaur skeleton">
```

Reply:
- [ ] According to the [step-by-step guide of giving a Code Review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate), it is always good to explain **why** you request a change.
    I think it will be better if you add the reason of this suggestion - it might be obvious for you but not for someone else. :no_good: